src/pudl/convert/ferc_to_sqlite.py

```diff
@@ -90,52 +73,50 @@ def parse_command_line(argv):
 
 def main():  # noqa: C901
     """Clone the FERC Form 1 FoxPro database into SQLite."""
+    load_dotenv()
     args = parse_command_line(sys.argv)
 
     # Display logged output from the PUDL package:
     pudl.logging_helpers.configure_root_logger(
         logfile=args.logfile, loglevel=args.loglevel
     )
 
-    with pathlib.Path(args.settings_file).open() as f:
-        script_settings = yaml.safe_load(f)
-
-    defaults = pudl.workspace.setup.get_defaults()
-    pudl_in = script_settings.get("pudl_in", defaults["pudl_in"])
-    pudl_out = script_settings.get("pudl_out", defaults["pudl_out"])
-
-    pudl_settings = pudl.workspace.setup.derive_paths(
-        pudl_in=pudl_in, pudl_out=pudl_out
-    )
-
-    parsed_settings = FercToSqliteSettings().parse_obj(
-        script_settings["ferc_to_sqlite_settings"]
-    )
-
-    # Configure how we want to obtain raw input data:
-    ds_kwargs = dict(
-        gcs_cache_path=args.gcs_cache_path, sandbox=pudl_settings.get("sandbox", False)
-    )
-    if not args.bypass_local_cache:
-        ds_kwargs["local_cache_path"] = Path(pudl_settings["pudl_in"]) / "data"
-
-    pudl_settings["sandbox"] = args.sandbox
-
-    if parsed_settings.ferc1_dbf_to_sqlite_settings:
-        pudl.extract.ferc1.dbf2sqlite(
-            ferc1_to_sqlite_settings=parsed_settings.ferc1_dbf_to_sqlite_settings,
-            pudl_settings=pudl_settings,
-            clobber=args.clobber,
-            datastore=Datastore(**ds_kwargs),
-        )
-
-    pudl.extract.xbrl.xbrl2sqlite(
-        ferc_to_sqlite_settings=parsed_settings,
-        pudl_settings=pudl_settings,
-        clobber=args.clobber,
-        datastore=Datastore(**ds_kwargs),
-        batch_size=args.batch_size,
-        workers=args.workers,
+    ferc_to_sqlite_job = ferc_to_sqlite.defs.get_job_def("ferc_to_sqlite_fast")
+    ferc_to_sqlite_job.execute_in_process(
+        run_config={
+            "resources": {
+                "ferc_to_sqlite_settings": {
+                    "config": EtlSettings.from_yaml(
+                        args.settings_file
+                    ).ferc_to_sqlite_settings.dict(
+                        exclude={
+                            "ferc1_dbf_to_sqlite_settings": {"tables"},
```

Comment:
Wish there was a less annoying way to do this... seems brittle when we add new tables. I guess we could potentially export the dict, then do some sort of recursive cleaning function to remove all `"tables"` keys... shrug. We can deal with it next time we add a FERC data source.

Reply:
I think if we want to remove the functionality to select a subset of tables, as @bendnorman mentioned below, we can just remove the `tables` from the settings and not have to worry about it all. I'd like others thoughts before just completely removing them though

Reply:
I'm ok with it!

Reply:
I like the idea of deprecating the table selection before we remove the settings property from the settings object.